### DISCUSSION TITLE
Last changed attribute on entities

### DISCUSSION THREAD

**Discussion by 1mfaasj** [kinghat/tabbed-card#122] [Q&A]
Hi, is it possible to include the last changed date attribute below the added entities?
Thanks

**Comment by ildar170975** [comment]
The `tabbed-card` does NOT contain entities.
It contains CARDS on each tab.
Whatever you want to display on a card - must be addressed to this card.

**Comment by 1mfaasj** [comment]
Yes I'm aware of the cards. I've got a card containing some sensors (card type entities) and want to show the last changed date to it, below the sensor itself 

Is that possible?

**Comment by ildar170975** [comment]
This question should not be asked here.
This is a GitHub repo - mainly for reporting bugs & FRs - for `tabbed-card` only.
Your question is NOT related to the `tabbed-card`.
I suggest you to close the issue and ask a question in HA community.

In short: last-changed may only be displayed in cards which support it "out-of-box" (like Entities card).
For cards which do NOT support displaying it - it still MAY be possible by using advanced technics like card-mod.

**Comment by kinghat** [comment]
ya this wouldnt have anything to do with tabbed-card but you could check on the home assistant community forums or their discord. ill move this to Q & A.
